linera-views/src/backends/memory.rs

```diff
@@ -111,6 +112,8 @@ static MEMORY_DATABASES: LazyLock<Mutex<MemoryDatabases>> =
 pub struct MemoryStore {
     /// The map used for storing the data.
     map: Arc<RwLock<MemoryStoreMap>>,
+    /// The `root_key`
```

Comment:
```suggestion
    /// The root key
```

(Either way not a very useful comment, though.